src/UI/Implementation/Component/Popover/Renderer.php

```diff
@@ -0,0 +1,82 @@
+<?php
+namespace ILIAS\UI\Implementation\Component\Popover;
+
+use ILIAS\UI\Implementation\Render\AbstractComponentRenderer;
+use ILIAS\UI\Implementation\Render\ResourceRegistry;
+use ILIAS\UI\Renderer as RendererInterface;
+use ILIAS\UI\Component;
+
+/**
+ * Class Renderer
+ *
+ * @author Stefan Wanzenried <sw@studer-raimann.ch>
+ * @package ILIAS\UI\Implementation\Component\Popover
+ */
+class Renderer extends AbstractComponentRenderer {
+
+	/**
+	 * @inheritdoc
+	 */
+	public function render(Component\Component $component, RendererInterface $default_renderer) {
+		$this->checkComponent($component);
+		$tpl = $this->getTemplate('tpl.popover.html', true, true);
+		$tpl->setVariable('FORCE_RENDERING', '');
+		/** @var Component\Popover\Popover $component */
+		$options = array(
+			'title' => $this->escape($component->getTitle()),
+			'placement' => $component->getPosition(),
+			'multi' => true,
+			'template' => str_replace('"', '\"', $tpl->get()),
+		);
```

Comment:
I like the stdClass better than an array for such purposes. However, maybe matter of personal pref. Array seems to work better with json_encode.